docs/testing.md

```diff
@@ -0,0 +1,45 @@
+## Testing toolchain configuration
```

Comment:
The idea is that Wix can set up a testing toolchain of junit and specs2, twitter of junit and scalatest and stripe only if scalatest?
Since unneeded rules won't be evaluated and the missing deps won't be a problem? If so it will probably be worthwhile to make sure that bazel toolchains actually works like that (for example move scalatest to this and see that Wix defines only specs2 + junit in the single testing toolchain and everything works).
Finally I think it's important in clarifying this idea.

Reply:
Toolchain only knows a list of labels, whether it correctly works depends on the actual rule that uses this toolchain and requests any of the dependencies. So if there is no rule that asks for JUnit, there's no need to add Junit deps to the toolchain.
Docs will be updated when there will be more rules that use it.

Reply:
I think there's value in explicitly specifying this issue of actual deps.
Most of our users will not understand toolchain in the same depths you do.